Homework1/task_2.py

```diff
@@ -0,0 +1,24 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. 
+# Сколько журавликов сделал каждый ребенок, если известно, что 
+# Петя и Сережа сделали одинаковое количество журавликов, 
+# а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+#     60 -> 10  40  10
+
+flag = True
+while flag:
+    s = int(input('Задайте количество журавликов: '))
+    if s % 6 != 0:
+        print('Задача не имеет решения. Количество журавликов должно делиться на 6.')
+    else:
+        flag = False
+     
+a = int(s / 6)
+b = int(4 * a)
```

Comment:
Тут лучше использовать целочисленное отделение.
```suggestion
a = s // 6
b = 4 * a
```